lib/plug/conn.ex

```diff
@@ -706,16 +772,23 @@ defmodule Plug.Conn do
 
   It is recommended for header keys to be in lower-case, to avoid sending
   duplicate keys in a request. As a convenience, this is validated during
-  testing which raises a `Plug.Conn.InvalidHeaderError` if the header key
-  is not lowercase.
+  testing where a header that is not lowercase raises a
```

Comment:
If we change the above section to match the section above the above section, could we also change this?